locales/en-US/notes.properties

```diff
@@ -77,6 +77,12 @@ savedComplete2=Saved at {time}
 # structure doesn't work for your locale, you can translate this as "Synced ({time})".
 syncComplete2=Synced at {time}
 
+# LOCALIZATION NOTE (syncComplete3): {time} is the time of syncing. Such as 5:33pm or 17:33.
+syncComplete3=Synced {time}
+
+# LOCALIZATION NOTE (syncToMail): {mail} is the email-id of the person currently logged in
+syncToMail=Sync to {mail}
```

Comment:
@ryanfeeley following @flodolo comment should we turn that into `Sync with {mail]`